android/java/org/chromium/chrome/browser/ntp/BraveNewTabPageLayout.java

```diff
@@ -922,12 +922,15 @@ private void initNews() {
             boolean isFeedLoaded = BraveActivity.getBraveActivity().isLoadedFeed();
             CopyOnWriteArrayList<FeedItemsCard> existingNewsFeedObject =
                     BraveActivity.getBraveActivity().getNewsItemsFeedCards();
-            int prevScrollPosition = SharedPreferencesManager.getInstance().readInt(
-                    Integer.toString(BraveActivity.getBraveActivity().getActivityTab().getId()));
-
-            mmViewedNewsCardsCount =
-                    SharedPreferencesManager.getInstance().readInt("mViewedNewsCardsCount_"
-                            + BraveActivity.getBraveActivity().getActivityTab().getId());
+            int prevScrollPosition = (BraveActivity.getBraveActivity().getActivityTab() != null)
```

Comment:
I would suggest move out `BraveActivity.getBraveActivity().getActivityTab()` and `BraveActivity.getBraveActivity().getActivityTab().getId()` into separate variables, as they are used more than once and the names are long.